src/components/workspace-utils.js

```diff
@@ -417,12 +417,12 @@ export const WorkspaceSubmissionStatusIcon = ({ status, loadingSubmissionStats,
       side: 'left'
     }, [
       Utils.switchCase(status,
-        ['success', () => icon('success-standard', { size, style: { color: colors.success() } })],
-        ['failure', () => icon('error-standard', { size, style: { color: colors.danger(0.85) } })],
-        ['running', () => icon('sync', { size, style: { color: colors.success() } })]
+        ['success', () => icon('success-standard', { size, style: { color: colors.success() }, 'aria-label': 'Workflow Status Success' })],
+        ['failure', () => icon('error-standard', { size, style: { color: colors.danger(0.85) }, 'aria-label': 'Workflow Status Failure' })],
+        ['running', () => icon('sync', { size, style: { color: colors.success() }, 'aria-label': 'Workflow Status Running' })]
```

Comment:
I would have thought that TooltipTrigger's useAsLabel should provide a label here. But with this aria-label, it seems like VoiceOver picks up the aria-describedby added by TooltipTrigger, and reads:
"Last Workflow Submitted Status" (column header)
"Workflow Status Success" (aria label)
"Last submitted workflow status: Success" (aria describedby)

Reply:
Yes, it's reading all 3. Slightly wordy, but better than nothing being read out.

Reply:
I was curious why it's read with aria-label but not aria-labelledby. Maybe something to do with aria-hidden on the icon.

Reply:
If I change `aria-label` to `aria-labelledby`, VoiceOver only reads the column header ("Last Workflow Submitted Status") and the tooltip ("Last submitted workflow status: Success"), but when I check with React Devtools, `aria-labelledby` is set to undefined. Should I use `aria-labelledby` instead of `aria-label`? 
<img width="322" alt="Screen Shot 2022-08-10 at 3 44 50 PM" src="https://user-images.githubusercontent.com/91842048/184007838-75e8d0e3-22ae-4395-8bb2-3201175e95dc.png">

```
Utils.switchCase(status,
        ['success', () => icon('success-standard', { size, style: { color: colors.success() }, 'aria-labelledby': 'Workflow Status Success' })],
        ['failure', () => icon('error-standard', { size, style: { color: colors.danger(0.85) }, 'aria-labelledby': 'Workflow Status Failure' })],
        ['running', () => icon('sync', { size, style: { color: colors.success() }, 'aria-labelledby': 'Workflow Status Running' })]
      )
```

Reply:
An `aria-labelledby` attribute should have the ID of another element that contains label text, not the label text itself. 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

I think this is fine as is. Up to you whether you want to dig into it or not.

Reply:
`aria-labelledby` would have to reference the ID of another component. For instance, if there was a visible label beside the icon, you could set the icon to be labelled by that other element. It is showing as undefined because the text you specified is not the ID of an element in the dom.

Reply:
Thanks, that makes sense!